src/components/HeroSection/HeroSection.module.scss

```diff
@@ -4,6 +4,9 @@
   align-items: center;
   padding: 165px 0 130px 0;
   background-color: $primary;
+  background-size: auto;
```

Comment:
Maybe `background-size: cover;` could be a more bulletproof option here?

Reply:
tried cover and contain, all have different drawbacks depending on screen size, but can try with cover

Reply:
fixed